QUARKUS-640.md

```diff
@@ -10,6 +10,7 @@ Following actions were taken to ensure test coverage and automation for reactive
  - Kafka consumer with JSON / AVRO body messages.
  - Kafka Streams with JSON body messages and windowed.
  - Tested over Kafka Strimzi and RH AMQ Streams.
+ - Kafka producer and consumer with batch processing
```

Comment:
RFE also mentions `Better rebalance handling` and `Checkpointing`

Reply:
I am not sure we can test better rebalance and Checkpointing is not yet implemented